.travis.yml

```diff
@@ -1,13 +1,22 @@
 language: php
 
 php:
-  - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
+matrix:
+  allow_failures:
+    - php: hhvm
+    - php: 7.0
+  fast_finish: true
+
 before_script:
-  - composer self-update
-  - composer install --no-interaction --prefer-source --dev
+  - curl --version
+
+install:
+  - travis_retry composer self-update
```

Comment:
I added it because there was a bug with curl in hhvm. But it seems a hhvm bug and not curl plugin related. Removed this now.